docs/tutorials/11_VarQTE.ipynb

```diff
@@ -0,0 +1,925 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Variational Quantum Time Evolution\n",
+    "This notebook demonstrates how to use Qiskit’s implementation of the Variational Quantum Time Evolution (VarQTE) algorithm for computing the time evolving state under a given Hamiltonian. Specifically, it introduces variational quantum imaginary and real time evolution based on McLachlan's variational principle, and shows how this can be leveraged using the `algorithms.time_evolvers.variational` sub-module."
```

Comment:
This notebook demonstrates how to use ~Qiskit’s implementation of~ the Variational Quantum Time Evolution (VarQTE)


I think we can just delete these words